evm/bridge-token-factory/contracts/BridgeTypes.sol

```diff
@@ -2,10 +2,10 @@
 pragma solidity ^0.8.24;
 
 library BridgeTypes {
-    struct FinTransferPayload {
-        uint128 nonce;
-        uint128 origin_nonce;
+    struct TransferMessagePayload {
```

Comment:
I personally liked FinTransferPayload

Reply:
Let's have consistent naming, so it will be easier to search it in the code. @olga24912 WDYT?

Reply:
I also like FinTransferPayload

Reply:
But it is actually an init transfer payload 😄